matlab/matlab-worker/src/main/java/org/ow2/proactive/scheduler/ext/matlab/worker/MatlabExecutable.java

```diff
@@ -131,43 +111,32 @@ public MatlabExecutable() {
     @Override
     public void init(final Map<String, Serializable> args) throws Exception {
 
-        // The tmp dir is matlab.task.tmpdir if defined otherwise it is java.io.tmpdir
-        this.tmpDir = System.getProperty(MATLAB_TASK_TMPDIR) == null ? System.getProperty("java.io.tmpdir")
-                : System.getProperty(MATLAB_TASK_TMPDIR);
+        logger.debug("Reading the global configuration");
 
-        // Fix for SCHEDULING-1308: With RunAsMe on windows the forked jvm can have a non-writable java.io.tmpdir
-        if (!new File(this.tmpDir).canWrite()) {
-            throw new RuntimeException("Unable to execute task, TMPDIR: " + this.tmpDir + " is not writable.");
-        }
-
-        // Read global configuration
         Object obj = args.get("global_config");
         if (obj != null) {
             this.paconfig = (PASolveMatlabGlobalConfig) obj;
         }
 
```

Comment:
use the Matlab configuration to dynamically set the logger(s) in debug mode. Do the same for scilab